### ISSUE TITLE
Download the available parameters

### ISSUE THREAD

**Issue by HtnrxUNc** [rockdaboot/wget2#244]
Hello.
I didn't find any documentation on the page about the use of "wget2". So I'm not sure that the parameters attached to it are the same as for "wget".
Example:
wget -m -np -t 0 -c [URL]
wget2 -m -np -t 0 -c [URL]
I'm not sure if the above two examples have the same effect, or if "wget2" inherits all the additional arguments from "wget"?
I should have defaulted to this idea, but what makes me unsure is that I found that after I ran "wget2 -m -np -t 0 -c [URL]".
"wget2" first creates a directory listing file (I'm not sure because I don't currently find any new files in my download directory) and then "wget2" updates the file listing of my target (download source address) on the screen.
So the same way of using "wget" applies to "wget2"? Or should I check the full "wget2" documentation?
Thank you for your review and reply.

**Comment by HtnrxUNc** [comment]
> Wget2 了解大多数 wget 选项。但是没有 FTP 支持。



> Wget2 understands most of the wget options. There is no FTP support, though.

Ok, thank you very much for your reply. I have found the file I need to download in my local directory.
(This may be different from "wget"
For example
When I run "wget -m -np http://a.com:80" with "wget", "wget" will store the file in the "a.com:80" directory of the current directory.
And when I use "wget2" to execute "wget2 -m -np http://a.com:80", "wget2" will store the file in the "a.com" directory of the current directory.
)
Maybe this difference made me start this question by mistake.
I now know that "wget2" uses most of the same parameters as "wget", with the exception of "FTP" support.
And I have successfully found the downloaded file.
Thank you very much for your answer. I will close this issue later.
I wish you a happy work and life.

Translated with www.DeepL.com/Translator (free version)

> Wget2 understands most of the wget options. There is no FTP support, though.

Ok, thank you very much for your reply. I have found the file I need to download in my local directory.
(This may be different from "wget"
For example
When I run "wget -m -np http://a.com:80" with "wget", "wget" will store the file in the "a.com:80" directory of the current directory.
And when I use "wget2" to execute "wget2 -m -np http://a.com:80", "wget2" will store the file in the "a.com" directory of the current directory.
)
Maybe this difference made me start this question by mistake.
I now know that "wget2" uses most of the same parameters as "wget", with the exception of "FTP" support.
And I have successfully found the downloaded file.
Thank you very much for your answer. I will close this issue later.
I wish you a happy work and life.

Translated with www.DeepL.com/Translator (free version)

**Comment by HtnrxUNc** [comment]
> Wget2 understands most of the wget options. There is no FTP support, though.

Ok, thank you very much for your reply. I have found the file I need to download in my local directory.
(This may be different from "wget"
For example
When I run "wget -m -np http://a.com:80" with "wget", "wget" will store the file in the "a.com:80" directory of the current directory.
And when I use "wget2" to execute "wget2 -m -np http://a.com:80", "wget2" will store the file in the "a.com" directory of the current directory.
)
Maybe this difference made me start this question by mistake.
I now know that "wget2" uses most of the same parameters as "wget", with the exception of "FTP" support.
And I have successfully found the downloaded file.
Thank you very much for your answer. I will close this issue later.
I wish you a happy work and life.

Translated with www.DeepL.com/Translator (free version)
